Assets/Scripts/Game/Questing/Components/SpecialNPCClickBehaviour.cs

```diff
@@ -32,6 +32,14 @@ namespace DaggerfallWorkshop.Game.Questing
     /// </summary>
     public class SpecialNPCClickHandler : MonoBehaviour
     {
+        StaticNPC npc; // needed for talk window setup
+
+        public StaticNPC StaticNPC
```

Comment:
Shouldn't need to store this here. StaticNPC is another MonoBehaviour-derived component peered with SpecialNPCClickHandler. As you already have the GameObject, just use something like:

StaticNPC npc = npcObject.GetComponent<StaticNPC>();

Or from within SpecialNPCClickHandler itself:

StaticNPC npc = this.GetComponent<StaticNPC>();

Just a heads-up, future of SpecialNPCClickHandler is a little uncertain. I'm happy with this here for now, but wouldn't want to build out this class too much. :)

Reply:
ok, will do so ;)